src/modules/dashboard/components/ActionsPage/ActionsPage.tsx

```diff
@@ -98,7 +98,7 @@ const ActionsPage = () => {
       loading: colonyActionLoading,
       error: colonyActionError,
     },
-  ] = useColonyActionLazyQuery();
+  ] = useColonyActionLazyQuery({ fetchPolicy: 'network-only' });
```

Comment:
We don't want this, as colonyAction data is super heavy. The one way of doing this is to create one more query, which return motion status, and adding it as dependency to this query. So whenever motion status will change, then this query will be updated. @rdig idea :) If you have some other ideas, feel free to use them!